### DISCUSSION TITLE
Invalid Markup when opening new instance of dnGREP.WPF.sln VS2019/22

### DISCUSSION THREAD

**Discussion by fade2gray** [dnGrep/dnGrep#711] [General]
I'm always presented with `Invalid Markup`  when opening a fresh instance of dnGREP.WPF.sln in both VS2019 and VS2022, but if I just run the debugger, the problem seems to rectify itself.

Is this expected behaviour?

![InvalidMarkup](https://user-images.githubusercontent.com/1332009/180468155-e90358f1-8957-4d20-95d3-2a40fcba3143.png)

**Comment by doug24** [comment]
Do this still happen if you change the configuration from Any CPU to x64?  I do all the dev with it set to x64.

**Reply by fade2gray** [reply]
Happens whether Any CPU, x64 or x86 is selected.

**Reply by doug24** [reply]
When you say "open a fresh instance" - do you mean you haven't built it yet?  Then yes, this is expected. The xaml has references to local code in four dnGrep libraries, and that code has to be built before the xaml designer can load it.  You don't need to run the debugger, just build the solution first.  The same error will show if you clean the solution and try to open the xaml designer.
